Sources/SPTDataLoaderResponse.m

```diff
@@ -63,6 +65,13 @@ - (instancetype)initWithRequest:(SPTDataLoaderRequest *)request response:(nullab
                                          userInfo:nil];
             }
             _responseHeaders = httpResponse.allHeaderFields;
+            NSString* value = [_responseHeaders valueForKey:SPTDataLoaderURLCacheSpecialHeader];
+            if (value && [value isEqualToString:@"YES"]) {
+                NSMutableDictionary *headers = [httpResponse.allHeaderFields mutableCopy];
+                [headers removeObjectForKey:@"Cached"];
```

Comment:
Shouldn't be that one SPTDataLoaderURLCacheSpecialHeader ?

Reply:
oops. indeed. good catch